test/consumer/index-spec.js

```diff
@@ -74,27 +75,22 @@ describe('Consumer tests', function () {
     receiveQueueSize: 5,
     logLevel: LEVELS.INFO,
   });
+  consumers.push(
```

Comment:
Why not just declare consumers here instead of pushing?

Reply:
The consumers array was made so we don't have to infinitely add "if else" statements in the afterEach section.
I can move the new consumers I made up, but they are specifically used for the state change tests.